apps/site/src/app/(main)/portal/@applicant/VerticalTimeline.tsx

```diff
@@ -3,6 +3,7 @@ import Image from "next/image";
 import BorderCircle from "@/assets/icons/border-circle.svg";
```

Comment:
Technically out of scope but could you change the color of `CheckCircle` to `green` instead of `lightgreen`?